dandi/dandiapi.py

```diff
@@ -649,17 +649,17 @@ def check_schema_version(self, schema_version: str | None = None) -> None:
             schema_version = models.get_schema_version()
         server_info = self.get("/info/")
         server_schema_version = server_info.get("schema_version")
-        if not server_schema_version:
-            raise RuntimeError(
-                "Server did not provide schema_version in /info/;"
-                f" returned {server_info!r}"
-            )
-        if server_schema_version != schema_version:
-            raise SchemaVersionError(
-                f"Server requires schema version {server_schema_version};"
-                f" client only supports {schema_version}.  You may need to"
-                " upgrade dandi and/or dandischema."
-            )
+        # if not server_schema_version:
```

Comment:
why this block is commented out -- doesn't LINC provide `/info/` with `schema_version`?

Reply:
When I was testing, my forked branch is technically tagged as a version of `0+untagged<some-sha-here>` thus I commented this out for testing purposes.

As noted in https://github.com/dandi/dandi-cli/pull/1519#issuecomment-2471300984 -- this will be commented back in once you et. al are happy with the core changes here